### PR TITLE
[DOCS] Code examples for transitionTo, replaceWith and recognize methods of Router service

### PR DESCRIPTION
**Pull request by sdebarros**
Part of [#16518](https://github.com/emberjs/ember.js/issues/16518)

### CODE REVIEW

**Comment by locks** [comment]
Thanks, will review soon!

File: packages/@ember/-internals/routing/lib/services/router.ts [hunk 1]
```diff
@@ -245,6 +277,27 @@ export default class RouterService extends Service {
      by the URL. Returns `null` if the URL is not recognized. This method expects to
      receive the actual URL as seen by the browser including the app's `rootURL`.
 
+     See [RouteInfo](/ember/release/classes/RouteInfo) for more info.
+
+     In the following example `recognize` id used to verify if a path belongs to our
```
Comment (rwjblue) [inline packages/@ember/-internals/routing/lib/services/router.ts]:
```suggestion
     In the following example `recognize` is used to verify if a path belongs to our
```

**Review by rwjblue — COMMENTED** [review]
One minor change, but otherwise LGTM

**Review by rwjblue — APPROVED** [review]

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**